lib/mongo.js

```diff
@@ -69,35 +69,37 @@ class Mongo {
   async getPackagesWithMaintainersAndRevenue () {
     this.log.info('Retrieving all packages with maintainers and revenue from db')
 
-    return this.db.collection(PACKAGES_COLLECTION).find({
-      $and: [
-        {
-          maintainers: {
-            $ne: null
-          },
+    return this.db.collection(PACKAGES_COLLECTION).aggregate([
```

Comment:
i thought agg functions were slower than queries?

Reply:
don't think that's a general rule, but definitely not this one, since this one filters out the unwanted array elements before returning it